Captain-AJ.md

```diff
@@ -0,0 +1,54 @@
+# name: "@Captain-AJ"
+# project: "An autonomous butler robot"
+
+# Project Name
+A.L.F.R.E.D. Butlerbot
+
+## Summary
+
+I plan to build an Arduino controlled autonomous robot butler named A.L.F.R.E.D. (Advanced, Little, Fit, Robot, Education, Device) that can be summoned at the press of a 
+button and deliver items. I am excited because this practical project has been on my list for sometime now. 
```

Comment:
when you say press of a button do you mean you plan on programming an application to control your robot?

Reply:
Yes, I plan to make an application with manual and autonomous control options using PyQT5. One would make the robot move to points of reference using the tracking system (Pixy2 and OpenCV).

Reply:
cool! could you please add this under your plan and elaborate a little more on what that application would look like?